.github/workflows/pkg-tests.yml

```
@@ -77,3 +87,68 @@ jobs:
           -v $PWD:/home/workspace
           ${{ needs.extract.outputs.container }}
           /bin/sh -c "PKG_NAME=${{ matrix.package }} /home/workspace/pkg-tests.sh -A"
+
+      - name: "Get job status"
+        id: job-status
+        if: always()
```

Although this probably works, the officially supported way of doing it is as follows, I think:
```suggestion
        if: ${{ always() }}
```

---

Ok, I change this.